src/dialogsearch.cpp

```diff
@@ -21,6 +22,13 @@ DialogSearch::DialogSearch(QWidget *parent) :
     ui->list_result->setSelectionBehavior(QAbstractItemView::SelectRows);
     ui->list_result->setEditTriggers(QAbstractItemView::NoEditTriggers);
 
+
+    for (auto *box : {ui->combo_variable, ui->combo_switch, ui->combo_item, ui->combo_eventname})
+    {
+        box->completer()->setFilterMode(Qt::MatchContains);
+        box->completer()->setCompletionMode(QCompleter::PopupCompletion);
+    }
+
```

Comment:
This will lead to
`ui->groupBox->findChildren<QComboBox*>()`
because the dialog itself does not inherit from QObject (strange, isn't it?!) I could do that, should I?